js/feed/createPost.mjs

```diff
@@ -0,0 +1,40 @@
+import { postMethod } from "../imports/request-methods/post.mjs";
+import { urlPosts } from "../imports/url.mjs";
+
+const title = document.querySelector("#post_title");
+const body = document.querySelector("#post_body");
+const image = document.querySelector("#post_image");
+
+const postHTML = document.querySelector("#post_container");
+
+async function newPost() {
+    const postData = {
+        title: title.value,
+        body: body.value,
+        media: image.value
+    };
+    
+    try {
+        const postRequest = await postMethod(postData);
+        const response = await fetch(urlPosts, postRequest);
```

Comment:
Could you check if the response is valid or not? If it is invalid, you have to throw an error message and handle the error in the UI.